pkg/vmimage/data/script.sh

```diff
@@ -164,7 +172,29 @@ yum -y install \
     tree \
     find \
     WALinuxAgent \
-    yum-utils
+    yum-utils \
+    azure-mdsd \
+    azure-security \
+    azsec-monitor \
+    td-agent
+
+# td-agent needs access to the journal
+usermod -a -G systemd-journal td-agent
+
+# openshift-audit dir ends up with wrong permissions, preemptively fix that
```

Comment:
That sounds like a umask issue, should we just be setting that? Also, do we want the audit log dir to be world-readable? That doesn't seem right to me.

Reply:
not a umask issue - https://github.com/openshift/origin/blob/a1e38f8c0d105eb1eed897de5ac46f2bbef832a9/vendor/gopkg.in/natefinch/lumberjack.v2/lumberjack.go#L191

0750 and a chgrp would probably be best

Reply:
btw, both openshift and fluentd run as root, so no chgrp needed, I did chmod 0750